README.md

```diff
@@ -18,6 +18,21 @@ yarn create t3-app
 # or
 pnpm create t3-app@latest
 ```
+To start your app 
+```bash
+yarn
+yarn dev
+```
```

Comment:
Please remove this section

Reply:
Resolved in 44fd30a35613cea5114750ee12218241712c7c13